### ISSUE TITLE
New translation: Czech

### ISSUE THREAD

**Issue by hason** [squidfunk/mkdocs-material#809]
``` jinja
{% macro t(key) %}{{ {
  "language": "cs",
  "clipboard.copy": "Kopírovat do schránky",
  "clipboard.copied": "Zkopírováno do schránky",
  "edit.link.title": "Upravit tuto stránku",
  "footer.previous": "Předchozí",
  "footer.next": "Další",
  "meta.comments": "Komentáře",
  "meta.source": "Zdroj",
  "search.language": "cs",
  "search.placeholder": "Hledat",
  "search.result.placeholder": "Pište co se má vyhledat",
  "search.result.none": "Nenalezeny žádné dokumenty",
  "search.result.one": "Nalezený dokument: 1",
  "search.result.other": "Nalezené dokumenty: #",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Přeskočit obsah",
  "source.link.title": "Přejít do repozitáře",
  "toc.title": "Obsah"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Great, thanks!

**Comment by squidfunk** [comment]
Released as part of 2.9.2. Thanks for your contribution!
